core/src/main/java/ar/com/wolox/wolmo/core/util/ImageUtils.java

```diff
@@ -0,0 +1,220 @@
+package ar.com.wolox.wolmo.core.util;
+
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.provider.MediaStore;
+import android.support.annotation.IntRange;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringDef;
+import android.support.annotation.StringRes;
+import android.support.v4.app.Fragment;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.lang.annotation.Retention;
+
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+public class ImageUtils {
+
+    @Retention(SOURCE)
+    @StringDef({
+            PNG,
+            JPG
+    })
+    public @interface ImageFormat {
+    }
+
+    public static final String PNG = "png";
+    public static final String JPG = "jpg";
+
+    private ImageUtils() {
```

Comment:
Minor thing, but we can put both curly braces together if there is no implementation and no comments. :smile: